script.sh

```diff
@@ -66,7 +50,8 @@ fi
 echo "stylelint version: $(npx --no-install -c 'stylelint --version')"
 
 echo '::group:: Running stylelint with reviewdog 🐶 ...'
-__run_stylelint | __filter_json | __run_reviewdog
+stylelint_results=$(__run_stylelint)
+echo "${stylelint_results}" | __run_reviewdog
```

Comment:
This may seem redundant. However, it fixes an error raised by Stylelint when running Reviewdog with the 'local' reporter:
> node:events:492
>       throw er; // Unhandled 'error' event
>       ^
> 
> Error: write EPIPE
>     at afterWriteDispatched (node:internal/stream_base_commons:160:15)
>     at writeGeneric (node:internal/stream_base_commons:151:3)
>     at Socket._writeGeneric (node:net:952:11)
>     at Socket._write (node:net:964:8)
>     at writeOrBuffer (node:internal/streams/writable:399:12)
>     at _write (node:internal/streams/writable:340:10)
>     at Writable.write (node:internal/streams/writable:344:10)
>     at file:///Users/bmccutchen/InfoTech/mongoose/node_modules/stylelint/lib/cli.mjs:562:19
> Emitted 'error' event on Socket instance at:
>     at emitErrorNT (node:internal/streams/destroy:151:8)
>     at emitErrorCloseNT (node:internal/streams/destroy:116:3)
>     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
>   errno: -32,
>   code: 'EPIPE',
>   syscall: 'write'
> }